dataprofiler/profilers/profile_builder.py

```diff
@@ -825,3 +826,159 @@ def tqdm(l):
+                
+            except Exception as e:
+                utils.warn_on_profile('data_labeler', e)
+                self.options.set({'data_labeler.is_enabled': False})
+                
+        # Restore data labelers for all columns
+        for key in self._profile:
+
+            val = self._profile[key]
+            use_data_labeler = True
+            if val.options and isinstance(val.options, StructuredOptions):
+                use_data_labeler = val.options.data_labeler.is_enabled
+
+            if use_data_labeler:
+                data_labeler_profile = val.profiles['data_label_profile'] \
+                                          ._profiles['data_labeler']
+                data_labeler_profile.data_labeler = None
+                
+                if val.options and val.options.data_labeler.data_labeler_object:
+                    data_labeler_profile.data_labeler = val.options \
+                                                           .data_labeler \
+                                                           .data_labeler_object
+
+                if data_labeler_profile.data_labeler is None:
+                    data_labeler_dirpath = None
+                    if val.options:
+                        data_labeler_dirpath = val.options.data_labeler \
+                                                  .data_labeler_dirpath
+                    if key in data_labelers:
+                        data_labeler_profile.data_labeler = data_labelers[key]
+                    else:
+                        data_labeler_profile.data_labeler = DataLabeler(
+                            labeler_type='structured',
+                            dirpath=data_labeler_dirpath,
+                            load_options=None)
+
+    def save(self, filepath="profile.pkl"):
+        """
+        Save profiler to disk
+        
+        :param filepath: Path of file to save to
+        :type filepath: String
+        :return: None
+        """
+        # Delete data labelers as they can't be pickled
+        data_labelers = self._remove_data_labelers()
+
+        # Create dictionary for all metadata, options, and profile 
+        data = { 
+                "total_samples": self.total_samples,
+                "encoding": self.encoding,
+                "file_type": self.file_type,
+                "row_has_null_count": self.row_has_null_count,
+                "row_is_null_count": self.row_is_null_count,
+                "hashed_row_dict": self.hashed_row_dict,
+                "_samples_per_update": self._samples_per_update,
+                "_min_true_samples": self._min_true_samples,
+                "options": self.options,
+                "_profile": self.profile
+               } 
+
+        # Pickle and save profile to disk
+        with open(filepath, "wb") as outfile:
+            pickle.dump(data, outfile)
+
+        # Restore all data labelers
+        self._restore_data_labelers(data_labelers)
```

Comment:
@AnhTruong @JGSweets - here is an implementation that "swaps" data labelers rather than deleting and reloading them. Since I already had the code written out for working with multiple data labelers, I left the same logic in place just in case that behavior changes in the future.